### DISCUSSION TITLE
Courses to add - 2022 2H

### DISCUSSION THREAD

**Discussion by kimpro82** [kimpro82/MOOCoke#52] [Ideas]
continuous from [Courses to add - 2022 1H #3](https://github.com/kimpro82/MOOCoke/discussions/3)

**Comment by kimpro82** [comment]
## Coursera / Azure

### Professional Certificates

Microsoft Azure Data Engineering Associate (DP-203) Professional Certificate
https://www.coursera.org/professional-certificates/microsoft-azure-dp-203-data-engineering

Microsoft Azure Developer Associate (AZ-204) Professional Certificate
https://www.coursera.org/professional-certificates/azure-developer-associate

Microsoft Azure Data Scientist Associate (DP-100) Professional Certificate
https://www.coursera.org/professional-certificates/azure-data-scientist

### Specialization

Microsoft Azure Fundamentals AZ-900 Exam Prep Specialization
https://www.coursera.org/specializations/microsoft-azure-fundamentals-az-900

Microsoft Azure Data Fundamentals DP-900 Exam Prep Specialization
https://www.coursera.org/specializations/microsoft-azure-dp-900-data-fundamentals

Microsoft Azure AI Fundamentals AI-900 Exam Prep Specialization
https://www.coursera.org/specializations/microsoft-azure-ai-900-ai-fundamentals

### Course

Cloud Computing Basics (Cloud 101)
https://www.coursera.org/learn/cloud-computing-basics

Getting Started with Azure
https://www.coursera.org/learn/cloud-azure-intro

Azure Infrastructure Fundamentals
https://www.coursera.org/learn/azure-infrastructure

Developing AI Applications on Azure
https://www.coursera.org/learn/developing-ai-applications-azure

Data Processing with Azure
https://www.coursera.org/learn/data-processing-with-azure

**Reply by kimpro82** [reply]
Add : Coursera Azure #59

**Comment by kimpro82** [comment]
## Coursera / ~~Mathematics~~ integrate into DS_MATLAB


### Specialization

Algebra: Elementary to Advanced Specialization
https://www.coursera.org/specializations/algebra-elementary-to-advanced

~~Mathematics for Machine Learning Specialization~~ DS_Python
https://www.coursera.org/specializations/mathematics-machine-learning

~~Mathematics for Engineers Specialization~~ already included in DS_MATLAB
https://www.coursera.org/specializations/mathematics-engineers

Expressway to Data Science: Essential Math Specialization
https://www.coursera.org/specializations/expressway-to-data-science-essential-math


### Course

Introduction to Calculus
https://www.coursera.org/learn/introduction-to-calculus

Applied Calculus with Python
https://www.coursera.org/learn/applied-calculus-with-python

**Reply by kimpro82** [reply]
Add : Coursera Mathematics #61

**Comment by kimpro82** [comment]
## Coursera / Statistics

Statistics with Python Specialization
https://www.coursera.org/specializations/statistics-with-python

Data Science: Statistics and Machine Learning Specialization
https://www.coursera.org/specializations/data-science-statistics-machine-learning

Advanced Statistics for Data Science Specialization
https://www.coursera.org/specializations/advanced-statistics-data-science

Bayesian Statistics Specialization
https://www.coursera.org/specializations/bayesian-statistics

Business Statistics and Analysis Specialization
https://www.coursera.org/specializations/business-statistics-analysis

**Reply by kimpro82** [reply]
Add : Coursera Statistics #63

**Comment by kimpro82** [comment]
## Coursera / GCP (Professional Certificates, Beginner)

[**Google Cloud Digital Leader Training**](https://www.coursera.org/professional-certificates/google-cloud-digital-leader-training)

[**Preparing for Google Cloud Certification: Cloud DevOps Engineer**](https://www.coursera.org/professional-certificates/sre-devops-engineer-google-cloud)

**Reply by kimpro82** [reply]
Add : Coursera Google Cloud (Professional Certificates) #65

**Reply by kimpro82** [reply]
Separate GCP and Azure table from Cloud

**Comment by kimpro82** [comment]
## Coursera / Electronics (Specialization)

Electrodynamics Specialization
https://www.coursera.org/specializations/electrodynamics

Introduction to Electricity and Magnetism Specialization
https://www.coursera.org/specializations/introduction-to-electricity-magnetism

Semiconductor Devices Specialization
https://www.coursera.org/specializations/semiconductor-devices

FPGA Design for Embedded Systems Specialization
https://www.coursera.org/specializations/fpga-design

~~Embedding Sensors and Motors Specialization~~
~~https://www.coursera.org/specializations/embedding-sensors-motors~~

**Reply by kimpro82** [reply]
Add : Coursera Circuit #67

**Comment by kimpro82** [comment]
## ETC / Classic Languages

Fortran, Pascal, Algol, ……

**Reply by kimpro82** [reply]
Add : ETC / Classic Languages #69

**Comment by kimpro82** [comment]
## Coursera / Dart & Flutter

**Reply by kimpro82** [reply]
[Add : Coursera Dart & Flutter #71](https://github.com/kimpro82/MOOCoke/issues/71)

**Comment by kimpro82** [comment]
## Coursera / GCP (Professional Certificates, Intermediate)

**Reply by kimpro82** [reply]
Add : Coursera GCP (Professional Certificates, Intermediate) #73

**Comment by kimpro82** [comment]
## Coursera / Electronics (Course)

Introduction to Electronics
https://www.coursera.org/learn/electronics

Linear Circuits 1: DC Analysis
https://www.coursera.org/learn/linear-circuits-dcanalysis

Linear Circuits 2: AC Analysis
https://www.coursera.org/learn/linear-circuits-ac-analysis

RF and millimeter-Wave Circuit Design
https://www.coursera.org/learn/rf-mmwave-circuit-design

Introduction to Semiconductor Devices 1
https://www.coursera.org/learn/semiconductor-1

Introduction to Semiconductor Devices 2
https://www.coursera.org/learn/semiconductor-2

VLSI CAD Part I: Logic
https://www.coursera.org/learn/vlsi-cad-logic

VLSI CAD Part II: Layout
https://www.coursera.org/learn/vlsi-cad-layout

**Reply by kimpro82** [reply]
Add : Coursera Electronics (Course) #75

**Comment by kimpro82** [comment]
## Coursera / Blockchain (Beginner)

**Reply by kimpro82** [reply]
Add : Coursera Blockchain #77

**Comment by kimpro82** [comment]
## Coursera / Fintech

**Reply by kimpro82** [reply]
Add : Coursera Fintech #79

**Comment by kimpro82** [comment]
## Coursera / GCP (Professional Certificates, Intermediate) - Data Science

**Reply by kimpro82** [reply]
Add : Coursera GCP - Data Science #81

**Comment by kimpro82** [comment]
## Boostcourse / Mobile · Web

https://www.boostcourse.org/

**Reply by kimpro82** [reply]
Add : Boostcourse Mobile · Web #83

**Comment by kimpro82** [comment]
## Boostcourse / Computer Science

**Reply by kimpro82** [reply]
Add : Boostcourse - Computer Science #85

**Comment by kimpro82** [comment]
## Boostcourse / Data Science

**Reply by kimpro82** [reply]
Add : Boostcourse - Data Science #87

**Comment by kimpro82** [comment]
## Boostcourse / AI

**Reply by kimpro82** [reply]
Add : Boostcourse - AI [#89](https://github.com/kimpro82/MOOCoke/issues/89)

**Comment by kimpro82** [comment]
## Coursera / Fitness

**Reply by kimpro82** [reply]
Add : Coursera Health #91

**Comment by kimpro82** [comment]
## Coursera / Parenting

Newborn Baby Care Specialization
https://www.coursera.org/specializations/newborn-baby-care

Global Quality Maternal and Newborn Care
https://www.coursera.org/learn/global-quality-maternal-and-newborn-care

Stanford's Short Course on Breastfeeding
https://www.coursera.org/learn/breastfeeding

Nutrition and Lifestyle in Pregnancy
https://www.coursera.org/learn/nutrition-pregnancy

**Reply by kimpro82** [reply]
Add : Coursera Parenting #93

**Comment by kimpro82** [comment]
## Coursera / Music (except "production" "business")

**Reply by kimpro82** [reply]
Add : Coursera Music (1) #95

**Comment by kimpro82** [comment]
## Coursera / Music (including "production" "business")

**Reply by kimpro82** [reply]
Add : Coursera Music (2) #97

**Reply by Chuev-hub** [reply]
Hi! who are u

**Reply by kimpro82** [reply]
@Chuev-hub Did you ask me "who"? I'm just a learner in Korea with investigating MOOC courses. God bless Ukraine!
